src/briefcase/windows_icon.py

```diff
@@ -0,0 +1,92 @@
+from ctypes import *
+from win32ctypes.pywin32 import win32api
+
+
+# Structures in ico file
+class IconHeader(Structure):
+    _fields_ = [
+        ("Width", c_byte),
+        ("Height", c_byte),
+        ("Colors", c_byte),
+        ("Reserved", c_byte),
+        ("Planes", c_ushort),
+        ("BitsPerPixel", c_ushort),
+        ("ImageSize", c_int),
+        ("ImageOffset", c_int),
+    ]
+
+
+class GroupIcon(Structure):
+    _fields_ = [
+        ("Reserved", c_ushort),
+        ("ResourceType", c_ushort),
+        ("ImageCount", c_ushort),
+        # ("Enries", PIconDirResEntry),
+    ]
+
+
+# Structure for updating resource
+class IconDirResEntry(Structure):
+    _fields_ = [
+        ("Width", c_byte),
+        ("Height", c_byte),
+        ("Colors", c_byte),
+        ("Reserved", c_byte),
+        ("Planes", c_ushort),
+        ("BitsPerPixel", c_ushort),
+        ("ImageSize", c_int),
+        ("ResourceID", c_ushort),
+    ]
+
+
+RT_ICON = 3
+RT_GROUP_ICON = 14
+
+
+def apply_icon(icon, dest):
+    """
+    Use winapi UpdateResource to copy icon into exe
+
+    :param str icon: .ico file path
+    :param str dest: path of the .exe to update.
+    """
+
+    print("Applying icons from %s to %s", (icon, dest))
+
+    with open(icon, 'rb') as iconfile:
+        # header = GroupIcon()
+        group_header = GroupIcon.from_buffer_copy(iconfile.read(sizeof(GroupIcon)))
+        if group_header.ResourceType != 1:
```

Comment:
At column 65: (F405) 'sizeof' may be undefined, or defined from star imports: ctypes